codespell_lib/data/dictionary.txt

```diff
@@ -7365,7 +7378,9 @@ crowkay->croquet
 crowm->crown
 crrespond->correspond
 crsytal->crystal
+crsytallisation->crystallisation
```

Comment:
crsytallise->crystallise ?
(and also cyrstallise->crystallise further down)

I guess you could also add the -ize and -ization variants too, if you felt like it :us: 
And there's also 'crystalline' if you wanted even more variants! :rofl:

Reply:
All done